.drone.yml

```diff
@@ -36,6 +36,21 @@ steps:
       exclude:
       - pull_request
 
+- name: publish
+  image: maven:3.8-openjdk-17
+  depends_on:
+  - sonar
+  environment:
+    ARTIFACTORY_TOKEN:
+      from_secret: artifactory_token
+  commands:
+  - mvn clean install
+  - mvn -s ./jparest_settings.xml -f pom.xml install -B org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy -DuniqueVersion=false -DaltReleaseDeploymentRepository=artifactory::default::https://artifactory.digital.homeoffice.gov.uk/artifactory/callisto-jparest -DskipTests
+  when: 
+    event:
+     exclude:
+     - pull_request
```

Comment:
Doesn't this mean that we are publishing on every push to any branch without sorting out the versioning

Reply:
I have addressed this by limiting the push trigger to only trigger the web hook on the main branch.